docs/.vuepress/theme/styles/index.styl

```
@@ -216,15 +216,25 @@ a {
   overflow-x: scroll;
 }
 
+table.htCore {
+  display: table;
+}
+
 table {
   width: 100%;
-  display: table;
-  overflow: scroll;
 
   tr {
       border-top: none;
   }
 
+  td {
+      white-space: normal;
+
+      @media (max-width: $MQNarrow) {
+        white-space: nowrap;
+      }
+  }
```

Just a minor styling change.

```suggestion
  td {
    white-space: normal;

    @media (max-width: $MQNarrow) {
      white-space: nowrap;
    }
  }
```

---

Fixed within https://github.com/handsontable/handsontable/pull/9926/commits/618545d12ff033e16dbcd0fad961da0a90b1ca07